canonical-web-and-design/jaas-dashboard
# Persist the connection for the model watcher when viewing the model details

## Done

- When viewing the model details pages persist the same instance of the model watcher connection.
- Disconnect the model watcher connection when routing away from a model details page.
- The routing needed to be nested so that we have a persistent view for all of the model details pages as a place to put the watcher management code.

## QA

- Open the network tab and view the model list, once it goes through all of your models then clear the network list and view a model.
- It should connect the watcher, click the element in the network pane to see the frames.
- Navigate away from the model details view and you should see the connection terminate.
